modules/welectBidAdapter.js

```diff
@@ -0,0 +1,106 @@
+import * as utils from '../src/utils.js';
```

Comment:
Can you please update this to import just the needed functions? We recently removed all the 'utils.' roots from adapters and modules.

Reply:
Yes will do, thank you